templates/base.html

```diff
@@ -63,18 +63,45 @@ <h1>{{ config['APP_TITLE'] }}</h1>
 
     <script src="/static/excel-bootstrap-table-filter-bundle.js"></script>
 
-
+    <script type="text/javascript">
+        highlight = function () {
+            var discussed = "Discussed";
+            var ready = "Ready";
+            var proposed = "Proposed";
+
+            $.each($(".table tbody").find("tr"), function () {                    
+            if ($(this).text().toLowerCase().replace(/\s+/g, '').indexOf(discussed.replace(/\s+/g, '').toLowerCase()) == -1){
+            
+            } else{
+                $(this).css('background-color','orange');
```

Comment:
I know I asked for orange and green but now that I see them, they seem a bit bright lol. 

Looking for alternatives here: https://www.w3schools.com/tags/ref_colornames.asp

Let's try with "Linen" for discussed, and "PaleGreen" for ready ?

Reply:
colours are not great on my monitor. I have made the changes

Reply:
I honestly can't see the difference between "white" and "linen" on my monitor. It's possible others might have old LCD screens like me - trying "moccasin", it's very similar, but a bit darker. I will make a new pull request with the changes

Reply:
OK, looks good 👍